app/views/layouts/application.html.erb

```diff
@@ -104,15 +104,21 @@
       </div>
     <% end %>
     <%= yield %>
+    <% if FeatureFlag.enabled?(:runtime_banner) || ENV['E2E'] == 'true' %>
+      <div class="Browser-iOS">
+        <div class="runtime-banner">
+          <a href="<%= URL.deep_link(request.path) %>" target="_blank" class="flex items-center flex-1">
+            <%= inline_svg_tag("logo-forem-app.svg", aria: true, class: "crayons-icon crayons-icon--default", width: 32, height: 32, title: "Forem App icon") %>
```

Comment:
```suggestion
            <%= inline_svg_tag("logo-forem-app.svg", aria_hidden: true, class: "crayons-icon crayons-icon--default", width: 32, height: 32) %>
```

nitpick (blocking) - I feel like this icon is purely decorative, in which case we don't want to surface it to screen reader users (otherwise the link is read out as "Forem App icon, Open with the Forem app...")